packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -29,6 +29,7 @@
 
   .main-container {
     flex-grow: 1;
+    min-height: 100vh;
```

Comment:
what does this one fix?

Reply:
without a min height on the container, the overlay wouldn't span the rest of the screen. so if there was very little content on the screen, the user would have to click inside of the content to close the nav instead of anywhere on the screen

Reply:
now that i think about it, 100% might be better than 100vh

Reply:
added the light background on the overlay for visibility on the screenshots:
with min-height: 100%
<img width="832" alt="image" src="https://user-images.githubusercontent.com/30411845/30223145-a36a5578-9497-11e7-9e39-78cb616864f7.png">

without min-height:
<img width="838" alt="image" src="https://user-images.githubusercontent.com/30411845/30223169-bf22b80a-9497-11e7-93a6-1cf9ec6b3a77.png">